coalib/misc/BuildManPage.py

```diff
@@ -40,7 +40,7 @@ class BuildManPage(Command):
         ('output=', 'O', 'output file'),
         ('parser=', None, 'module path to an ArgumentParser instance'
          '(e.g. mymod:func, where func is a method or function which return'
-         'an arparse.ArgumentParser instance.'),
+         'an ``argparse.ArgumentParser`` instance.'),
```

Comment:
I don't think that you have to put this occurrence into double backticks, since I don't think that it is anywhere been ReST-rendered... Or is it? @sils @jayvdb @AbdealiJK